docgen/src/guides/migration.md

```diff
@@ -26,3 +26,24 @@ var search = instantsearch({
   }
 });
 ```
+
+## The items are not sorted like before in the refinementList / menu / hierarchicalMenu widget
+
+We changed the default sort order of those widgets. This might have impacted your implementation
+if you didn't specify them originally. To change back the order use the `sortBy` configuration
+key.
+
+```javascript
+search.addWidget(
+  instantsearch.widgets.refinementList({
+    container: '#brands',
+    attributeName: 'brand',
+    sortBy: ['name:desc'],
```

Comment:
I was expecting the old behaviour as example here, `sortBy: ['isRefined', 'count:desc']`